rules/framework.bzl

```diff
@@ -458,6 +458,8 @@ def _apple_framework_packaging_impl(ctx):
     virtualize_frameworks = feature_names.virtualize_frameworks in ctx.features
     if virtualize_frameworks:
         framework_info = _get_virtual_framework_info(ctx, framework_files, compilation_context_fields)
+        if outputs.swiftmodule:
```

Comment:
nit:  Move it out of the if-else clause since we have the exactly same code snippet in the else clause (i.e., line 475-477).

Reply:
Sure, I think we can take a pass at how `swiftinfo` is built out too. I can send this over as a followup